test/SILGen/witness_accessibility.swift

```diff
@@ -0,0 +1,42 @@
+// RUN: %target-swift-frontend -Xllvm -sil-full-demangle -emit-silgen %s | %FileCheck %s
+
+public protocol P {
```

Comment:
@jrose-apple @huonw I invite you to marvel at the subtle complexity and delicious texture of this test case.